docs/Interoperability_tests_guide/Using_API_Jenkins_integration.md

```diff
@@ -0,0 +1,310 @@
+# Jenkins/GitLab integration with interop tests API
+
+If you would like to automate your interoperability tests, you can use the Coiote DM API and integrate it with a CI/CD environment like Jenkins or GitLab.
+Follow the guide below to learn how to configure the integration, run tests and summarize your test execution using these tools.
+
+!!! note
+    The following instruction is based on integration with Jenkins. To integrate with GitLab, you can follow the same steps, but with slight adjustments - for details, please see subsection on [GitLab](#gitlab-configure-and-run-pipeline).    
+
+## Prerequisites
+
+- An active Jenkins and GitLab account.
+- A Git project repository.
+- A working Coiote DM installation and a port for communication with the installation API.
+- A device registered in the platform (if the tests require the device to be registered).
+- A Coiote DM user with access to the device and the following permissions:
+    - `rest.base.paths.devices.POST`
+    - `rest.base.paths.devices.DELETE_id`
+    - `rest.base.paths.devices.GET`
+    - `rest.base.paths.devices.GET_id`
+    - `rest.base.paths.devices.PUT_id`
+    - `rest.base.paths.devices.find.details.GET`
+    - `rest.base.paths.protocolTests.schedule.device.POST_device`
+    - `rest.base.paths.protocolTests.report.device.POST_deviceId`
+
+## Setting up a Jenkins standard pipeline
+
+1. Upload a file with python script used to run test cases to your project repository:
+    - Edit the following python script where required to adjust it to your environment:
+       ```
```

Comment:
Can we embed the code snippet with code highlighting? Or for example use gist?

Reply:
added code highlight - pls take a look and see if it's enough

Reply:
yup, that's fine ;)